tests/files/poppunk_output/test_network_zip/network/network_component_38.graphml

```diff
@@ -0,0 +1,529 @@
+<graphml xmlns="http://graphml.graphdrawing.org/xmlns" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://graphml.graphdrawing.org/xmlns http://graphml.graphdrawing.org/xmlns/1.0/graphml.xsd">
```

Comment:
can you please add instructions for how these were generated into the tests/files/popppunk_output directory